controllers/contact-controller.js

```diff
@@ -270,6 +270,12 @@ const ContactController = Contact => {
   };
 
   const createContact = async (body, accountSid) => {
+    // if a contact has been already created with this taskId, just return it (idempotence on taskId). Should we use a different HTTP code status for this case?
```

Comment:
I don't think we should return a different code.  If a previous call succeeded, the fact that we've received this one means that the original response probably never made it back.  So we should return the same response we would if this was the first one, because for the client, it probably is.

Reply:
We can use HTTP 409 for Duplicate resource or resource already exists. However, we would need to change the frontend to handle that scenario. So I don't think that change is really important right now.